### ISSUE TITLE
Inconsistent document structures are returned with `find` and `findById`

### ISSUE THREAD

**Issue by xavierchow** [couchbaselabs/node-ottoman#620]
I have an `Order` model defined as follows,

```js
// Order schema
{
  user: { userId: { type: String } },
  status: { type: String, required: true },
};
```

in the bucket, it's persisted with `_type='Order'` as expected.
```js
// Document in the Couchbase bucket
{
  "user": {
    "userId": "111"
  },
  "status": "paid",
  "id": "03ff9b4d-cd7f-4d90-8153-7af59d0aafd9",
  "_type": "Order"
}
```

When I use `model.findById`, it returns the raw document from the bucket as above(that includes the `_type`), but if I use the `model.find({ status: 'paid' })`(underneath it's a N1QL query), it returns orders **without** `_type` field.

```
[
   {
      "id":"03ff9b4d-cd7f-4d90-8153-7af59d0aafd9",
      "status":"paid",
      "user":{
         "userId":"111"
      }
   }
]
```

Is there an option either applies to `find` or `findById` to make sure the results are consistent?
I know passing `select` might work for `mode.find` but it will be too tedious to pass a full set of fields on the schema.

Any suggestions?

**Comment by AV25242** [comment]
Hello @xavierchow  thanks for reaching out, it looks like an overlook on our part. I will let one of our developers address this. At this point i am going to label it as a bug and we go from there.

**Comment by ejscribner** [comment]
Hey @xavierchow. This has been fixed in the latest release of Ottoman v2.1.0, available now on NPM. Thanks!

**Comment by xavierchow** [comment]
it works perfectly, thank you all so much @ejscribner @gsi-alejandro 
closing.
